js/jquery.ntm.js

```diff
@@ -14,7 +14,8 @@
         spoilerButtonClickMaxX: 20,
         spoilerButtonClickMinY: 8,
         spoilerButtonClickMaxY: 24,
-        slideEffect: true
+        slideEffect: true,
+        postRender : function () { $.noop }

```

Comment:
this should be `postRender: $.noop`
If you check https://api.jquery.com/jquery.noop/ $.noop is a method - an empty function already.
It is to avoid recreation of empty functions. 

Quoting from documentation - "This is useful for plugin authors who offer optional callbacks; in the case that no callback is given, something like jQuery.noop could execute."

YOU are the plugin author btw :+1: